specs/sonarqube/sonarqube.spec

```diff
@@ -0,0 +1,182 @@
+
+%define _posixroot        /
+%define _root             /root
+%define _bin              /bin
+%define _sbin             /sbin
+%define _srv              /srv
+%define _home             /home
+%define _opt              /opt
+%define _lib32            %{_posixroot}lib
+%define _lib64            %{_posixroot}lib64
+%define _libdir32         %{_prefix}%{_lib32}
+%define _libdir64         %{_prefix}%{_lib64}
+%define _logdir           %{_localstatedir}/log
+%define _rundir           %{_localstatedir}/run
+%define _lockdir          %{_localstatedir}/lock/subsys
+%define _cachedir         %{_localstatedir}/cache
+%define _spooldir         %{_localstatedir}/spool
+%define _crondir          %{_sysconfdir}/cron.d
+%define _sysctldir        %{_sysconfdir}/sysctl.d
+%define _loc_prefix       %{_prefix}/local
+%define _loc_exec_prefix  %{_loc_prefix}
+%define _loc_bindir       %{_loc_exec_prefix}/bin
+%define _loc_libdir       %{_loc_exec_prefix}/%{_lib}
+%define _loc_libdir32     %{_loc_exec_prefix}/%{_lib32}
+%define _loc_libdir64     %{_loc_exec_prefix}/%{_lib64}
+%define _loc_libexecdir   %{_loc_exec_prefix}/libexec
+%define _loc_sbindir      %{_loc_exec_prefix}/sbin
+%define _loc_bindir       %{_loc_exec_prefix}/bin
+%define _loc_datarootdir  %{_loc_prefix}/share
+%define _loc_includedir   %{_loc_prefix}/include
+%define _loc_mandir       %{_loc_datarootdir}/man
+%define _rpmstatedir      %{_sharedstatedir}/rpm-state
+%define _pkgconfigdir     %{_libdir}/pkgconfig
+
+%define __ldconfig        %{_sbin}/ldconfig
+%define __service         %{_sbin}/service
+%define __touch           %{_bin}/touch
+%define __chkconfig       %{_sbin}/chkconfig
+%define __updalt          %{_sbindir}/update-alternatives
+%define __useradd         %{_sbindir}/useradd
+%define __groupadd        %{_sbindir}/groupadd
+%define __getent          %{_bindir}/getent
+%define __sysctl          %{_sbindir}/sysctl
+%define __systemctl       %{_bindir}/systemctl
+
+################################################################################
+
+%define sonar_name       sonar-application
+%define short_name       sonar
+%define sonar_prefix     %{_opt}/%{name}-%{version}
+
+%define service_user     sonarqube
+%define service_group    sonarqube
+
+################################################################################
+
+Summary:              SonarQube Continuous Inspection
+Name:                 sonarqube
+Version:              7.9.1
+Release:              0%{?dist}
+License:              LGPLv3
+Group:                System Environment/Daemons
+URL:                  https://www.sonarqube.org/
+
+Source0:              https://github.com/SonarSource/%{name}/archive/%{version}.tar.gz
+Source1:              %{name}.service
+Source2:              %{name}.properties
+Source3:              %{name}.sysctl
+
+BuildRoot:            %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires:        java-11-openjdk java-11-openjdk-devel
```

Comment:
We provide our own packages with JRE/JDK, which we use for all builds. If this package requires Java 11, you should add `jdk11` package as required dependency instead of `java-11-*`

Reply:
OK. I do not mind (16dceb4).

Are these packages stable? For example, I found `jdk11` only in `kaos-testing`.

Reply:
Yep. All these packages will be released as part of #213.